build.gradle

```diff
@@ -515,8 +516,7 @@ task buildTinyWithEnum(dependsOn: "mergeTiny", type: FileOutput) {
 		String[] argsPropose = [
 			mergedFile.getAbsolutePath(), // must use official jar
 			noEnum.getAbsolutePath(),
-			namedWithEnum.getAbsolutePath(),
-			"--writeAll"
```

Comment:
Why'd you remove this?

Reply:
The arg was added incorrectly before (before this took 3 args, but since we don't go thorough command line main it did not report the wrong number of args (4 instead of 3)). Now the 4th arg is the input jar mapping.

Reply:
Ah, I see. So it didn't do anything... I guess I forgot to merge a feature, hah